model/import/CsvItemImportHandler.php

```diff
@@ -72,8 +77,16 @@ public function import(
                 if (Report::TYPE_SUCCESS === $itemImportReport->getType()) {
                     $itemValidatorResults->setFirstItem($itemImportReport->getData());
 
+                    $logger->debug(sprintf('Tabular import: successful import of item from line %s', $lineNumber));
+
                     $successReportsImport++;
                 } else {
+                    $logger->error(sprintf(
```

Comment:
The same as here https://github.com/oat-sa/extension-tao-itemqti/pull/1760/files#r644031797